smart-contracts/assembly/contracts/FT/token.ts

```diff
@@ -187,6 +187,11 @@ export function transfer(binaryArgs: StaticArray<u8>): void {
  * @returns true if the transfer is successful
  */
 function _transfer(from: Address, to: Address, amount: u256): void {
+  assert(
+    from.toString() != to.toString(),
```

Comment:
There is no equality implementation for Address?

Reply:
You're right !